Sources/Valkey/Node/ValkeyRunningClients.swift

```diff
@@ -0,0 +1,111 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the valkey-swift project
+//
+// Copyright (c) 2025 the valkey-swift authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See valkey-swift/CONTRIBUTORS.txt for the list of valkey-swift authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+@usableFromInline
+struct ValkeyRunningClients<
```

Comment:
Can we add a `StateMachine` at the end please? Otherwise this sounds very ref typey. Approving anyway.